src/main/java/pong/Paddle.java

```diff
@@ -1,4 +1,24 @@
 package pong;
 
 public class Paddle {
+    private int x, y;
+
+    public Paddle(Player player){
+        y = PongFrame.HEIGHT/2;
+        if (player.equals(Player.HUMAN)){
+            //set human paddle to right side of board
```

Comment:
I say that this is the right side of the board but I am not sure what the origin of the frame actually is so I don't know if what I am thinking is correct.